File: src/models/Cupcake.ts
// To parse this data:
//
//   import { Convert } from "./file";
//
//   const cupcakes = Convert.toCupcakes(json);

export interface Cupcake {
  id: string
  flavor: string
  color: string
  description: string
  price: number
  picturePath: string
}

// Converts JSON strings to/from your types
export class Convert {
  public static toCupcakes(json: string): Cupcake[] {
    return JSON.parse(json)
  }

  public static cupcakesToJson(value: Cupcake[]): string {
    return JSON.stringify(value)
  }
}

File: netlify/functions/data/cupcake.go
package main

type Cupcake struct {
	ID          string `json:"id"`
	Flavor      string `json:"flavor"`
	Color       string `json:"color"`
	Description string `json:"description"`
	Price       int64  `json:"price"`
	PicturePath string `json:"picturePath"`
}
